### ISSUE TITLE
Time stamp not rolling over to 0 when max value is reached

### ISSUE THREAD

**Issue by hperrey** [tblum/jadaq#29]
When recording data with the VX1751 over a longer period, the time stamp increases gradually until reaching a maximum value. On the roll-over the time stamp becomes not 0 but a rather large value like 2177316752. This happens on each overflow. See in the figure below for a 30min run:

![a337fa6f7b7e4bedd6e22b5733e5731468060b7e](https://user-images.githubusercontent.com/2509713/58109344-c4290900-7bed-11e9-9b98-f4fc656fc9b5.png)

I suspect that this is signed/unsigned variable mismatch or that the width of the time stamp variable is not correct.

**Comment by fmessi** [comment]
No, you are reading one bit too much: the MSB is actually the overflow. 
We had the same issue with the V1740: once the counter reaches the limit the first bit does not reset to 0, acting as an overflow indicator... either you ignore it, or you have to reset “manually”

**Comment by hperrey** [comment]
Thanks, I thought I had seen this behavior not too long ago! :)
I think that calls for a fix within jadaq though.
